README.rst

```diff
@@ -98,45 +98,48 @@ Documentation (WIP)
 
 The `full documentation <https://serenata_toolbox.readthedocs.io>`_ is still a work in progress. If you wanna give us a hand you will need `Sphinx <http://www.sphinx-doc.org/>`_:
 
-::
+.. code-block:: bash
 
   $ cd docs
   $ make clean;make rst;rm source/modules.rst;make html
 
 Contributing
 ------------
 
-Within your `virtualenv <https://virtualenv.pypa.io/en/stable/>`_:
+Start by cloning the repository and installing the dependencies. You should create a development environment with `virtualenv <https://virtualenv.pypa.io/en/stable/>`_ by running:
```

Comment:
Actually as this is a `pip` installable project there's no _installing the dependencies_ step. The `python setup.py develop` takes care of that (and actually installing the dependencies otherwise can lead to issues in the development process).

Note that in the bash steps ahead there's no line to install the dependencies.

Reply:
I'll change that.